src/tools/sdk/_windows.md

```diff
@@ -1,23 +1,41 @@
+###How to install?
+
 You can install the Dart SDK using [Chocolatey.][Chocolatey]
 
-{{site.alert.important}}
-These commands require administrator privileges.
-If you need help on starting an administrator-level command prompt,
-try a search like
-<em><a href="https://www.google.com/search?q=cmd+admin"
-  target="blank">cmd admin</a>.</em>
-{{site.alert.end}}
+> **_NOTE:_** These commands require administrative privileges. To start command prompt as an administrator following these steps:
+> 
+> - Press Windows+R to open the “Run” box. 
+> - Type “cmd” into the box 
+> - Press Ctrl+Shift+Enter to run the command as an administrator.
```

Comment:
I agree that telling the user to search isn't the best solution, but I'm not sure this is as users may not want to use the old style run command prompt and may be using something like the new Windows Terminal.  I'll give some thought on how we can perhaps make this more generic.

Reply:
We can also add instructions for Windows Powershell which is the new Windows terminal. Although, only step 1 will change and rest will remain the same.